src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -79,7 +80,7 @@ function InboxReason({inbox}: Props) {
 
   return (
     <Tag type={tagType} tooltipText={tooltip}>
```

Comment:
Tag unfortunately sets font size on a span inside the tag

Reply:
Can we move the font size in Tag to the root component so that you can style `<Tag>` here instead of creating a wrapper?